libs/SalesforceSDK/src/com/salesforce/androidsdk/push/SFDCFcmListenerService.java

```diff
@@ -45,10 +45,8 @@
     @Override
     public void onMessageReceived(RemoteMessage message) {
         if (message != null && SalesforceSDKManager.hasInstance()) {
-            final PushNotificationInterface pnInterface = SalesforceSDKManager.getInstance().getPushNotificationReceiver();
-            if (pnInterface != null) {
-                pnInterface.onPushMessageReceived(message);
-            }
+            final PushNotificationDecryptor pnDecryptor = PushNotificationDecryptor.getInstance();
+            pnDecryptor.onPushMessageReceived(message);
```

Comment:
Sends the incoming message to the decryptor, which will then forward it to the interface once processing is complete.